### PR TITLE
Add Nashiro dataset to documentation

### PR DESCRIPTION
**Pull request by Kasambx**
I added the Nashiro dataset link for Heart rate variability biofeedback training and emotion regulation.

### CODE REVIEW

File: docs/multi-echo.rst [hunk 1]
```diff
@@ -578,15 +578,15 @@ This list is not necessarily up to date, so please check out OpenNeuro to potent
 * `Cambridge Centre for Ageing Neuroscience (Cam-CAN)`_
 * `rt-me-fMRI - A task and resting state dataset for real-time, multi-echo fMRI methods development and validation`_
 * `EuskalIBUR`_
-
+* Nashiro-Dataset_
```
Comment (handwerkerd) [inline docs/multi-echo.rst:581]:
```suggestion
* Nashiro-Dataset_ - Heart rate variability biofeedback training and emotion regulation
```

Very minor suggestion. 
Thank you for adding this!

**Review by handwerkerd — CHANGES_REQUESTED** [review]

**Review by Kasambx — COMMENTED** [review]
worked on the commit suggestion.

**Review by handwerkerd — APPROVED** [review]
Thank you for adding this!

**Review by tsalo — APPROVED** [review]
LGTM!

**Review by dowdlelt — APPROVED** [review]
I just used this portion of the FAQ the other day, extraordinarily useful. LGTM

**Comment by tsalo** [comment]
Thanks @Kasambx!

**Comment by Kasambx** [comment]
Its been a pleasure looking forward to contribute more to your projects.

**Pull request merged**